main.tf

```diff
@@ -47,7 +47,6 @@ resource "aws_ecs_task_definition" "default" {
   memory                   = "${var.task_memory}"
   execution_role_arn       = "${aws_iam_role.ecs_exec.arn}"
   task_role_arn            = "${aws_iam_role.ecs_task.arn}"
-  healthcheck              = "${var.healthcheck}"
```

Comment:
This should be passed to the container definition, not the task definition.

Reply:
https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definition_parameters.html#container_definition_healthcheck

Reply:
Yea sorry I added extra confusion here - however, we already do this:

https://github.com/cloudposse/terraform-aws-ecs-web-app/blob/master/main.tf#L42